rtloader/two/two.cpp

```diff
@@ -42,17 +43,23 @@ Two::Two(const char *python_home)
 
 Two::~Two()
 {
+    // For more information on why Py_Finalize() isn't called here please
+    // refer to the header file or the doxygen documentation.
     PyEval_RestoreThread(_threadState);
+    free(_pythonHome);
```

Comment:
technically, since we don't call `Py_Finalize` anymore, I don't know if we can actually free this safely...

Reply:
True, I will remove this from the destructors.

Reply:
This should fix it: https://github.com/DataDog/datadog-agent/pull/3788